docs/dcp2_schema_evolution_specs.rst

```diff
@@ -0,0 +1,158 @@
+
+.. role:: raw-html(raw)
+   :format: html
+.. |nn| replace:: :raw-html:`<blockquote>`
+.. |ne| replace:: :raw-html:`</blockquote>`
+
+==============================
+DCP2 Metadata Schema Evolution
+==============================
+
+.. contents::
+
+Overview
+========
+
+The HCA metadata standard is developed in a transparent and open manner so that the whole HCA community can participate
+in the process. It follows a set of `design principles`_ that serve as a foundation for the metadata standard in order
+to ensure it meets the needs of HCA data contributors, consumers and DCP downstream components.
+
+The metadata schemas are versioned following semantic `versioning rules`_ that account for backwards compatibility.
+However, compatibility of the schemas does not account for impact on the schema consumers, especially if different
+versions of the same schemas must be maintained, indexed, displayed and/or interacted with in any form.
+
+The metadata is updated through PRs in the `metadata-schema repository`_. The specific details can be found in the
+documentation in the repository, but the overall flow can be described as follows:
+
+- A ticket is opened describing an issue
+- A member of the Metadata Team looks into the issue, there is a discussion and, if deemed important, a PR is created
+  against the schema
+- The PR is reviewed within a time frame specified by the type of change (Major/Minor/Patch). The review includes
```

Comment:
This Major/Minor/Patch distinction is dangerously simplistic. For one, a logical schema change could affect different schema documents in different ways, and bump different parts of their semantic version numbers. More importantly, a minor, or even patch level change to a schema document could have major impacts downstream. You simply don't know when you make the change.

Reply:
Do you have a suggestion for a better way to categorise/version changes? As you point out, the metadata team doesn't have in-depth knowledge about how the changes we make affect DCP components or other consumers of our schema. The best we can do is version our schema based on how the schema itself changes and the compatibility between schema versions. We don't have control over how other DCP components or independent users of our schema interpret and make assumptions about the schema so I am not sure how we can factor these things into how we version the schema? Or are you saying the way we apply the major/minor/patch categorisations needs to be reviewed?

Reply:
We can continue to version the individual schemas as we do right now, I'm not objecting to that. I'm objecting to the assumption that a minor schema change has minor impact on downstream components. There is no easy way (and no need) to categorize schema changes by impact *before* the PR is made. The determination of impact is what the PR review process is all about.

I'm working on a less verbose and more precise version of this SOP which will make that clear.